src/components/ToDo.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import Button from "../elements/button/index.jsx";
+import Input from "./input/index.jsx";
+import TaskList from "./taskList/index.jsx";
+import '../App.css';
+import '@fortawesome/fontawesome-free/css/all.min.css';
```

Comment:
Lets add the import  `import '@fortawesome/fontawesome-free/css/all.min.css';` in the `App.jsx` file.
In this way the css styling will be available in all the children component and we won't need to import in each file separately.